contrib/win32/win32compat/w32-doexec.c

```diff
@@ -238,6 +243,28 @@ setup_session_env(struct ssh *ssh, Session* s)
 	return ret;
 }
 
+void do_motd_windows(int fd) {
+	if (options.print_motd) {
+		FILE* f;
+		char buf[256];
+		buf[0] = '\0';
+		if (check_secure_file_permission(_PATH_MOTD_FILE, NULL, 1) == 0) {
+			f = fopen(_PATH_MOTD_FILE, "r");
+			if (f) {
+				while (fgets(buf, sizeof(buf), f)) {
+					if (atomicio(vwrite, fd, buf, strlen(buf)) != strlen(buf))
```

Comment:
The formatting is messed up when I have multiple lines

<img width="662" alt="image" src="https://user-images.githubusercontent.com/23668037/145921159-8a39b3f9-e5ce-4b39-bdb7-d12c673d7abe.png">

Reply:
There is one more issue which might annoy the customers using MOTD is the screen is scrolled up. Because of this, MOTD is not shown to customers unless they scroll up. I believe this was conhost.exe behavior, please check with the console team https://github.com/DHowett if it's possible to revert that behavior.